doc/source/cluster/aws-tips.rst

```diff
@@ -5,39 +5,61 @@
 AWS Configurations
 -------------------
 
-.. _aws-cluster-efs:
+.. _aws-cluster-ec2:
 
-Using Amazon EFS
-~~~~~~~~~~~~~~~~
+Amazon EC2
+~~~~~~~~~~
+
+CPUs and GPUs
+=============
+
+Deploy Ray clusters on AWS using any combination of CPUs and GPUs.  
+
+
+.. code-block:: text
+
```

Comment:
Can you include the cluster name and provider fields so this becomes a fully copy/pastable example? (it should only add 3 lines to the config)